qa/integration-tests/src/test/java/io/camunda/it/rdbms/db/util/TestConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.rdbms.db.util;
+
+import javax.sql.DataSource;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+
+@Configuration
+@EnableAutoConfiguration
+public class TestConfiguration {
+
+  // TODO find out why we need to define this bean
```

Comment:
TODO cleared: Need to redefine this one because we excluded `DataSourceTransactionManagerAutoConfiguration` in application.properties

We also could add it as @Import to testconfiguration 🤔